hex/helpers/lib/parse_deps.exs

```diff
@@ -109,7 +109,7 @@ defmodule Parser do
     %{
       type: "git",
       url: repo_url,
-      branch: opts[:branch] || "master",
```

Comment:
@TylerWitt any idea why this might be failing? You seem to be a lot more knowledgable than me about Hex/Erlang... 

If we can figure out a fix, should probably split this out to it's own dedicated PR...

Reply:
Sorry for the delay!

This might have been a temporary hex outage or other transient error. I cherry-picked this commit and the build worked fine. I'd retry CI.

Reply:
No worries, thanks for trying--your timing is impeccable... you probably tried it just a few days after it was fixed 😁 :
* https://github.com/dependabot/dependabot-core/pull/12109